### PR TITLE
chore: misc style changes

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

N/A

# Description ℹ️

- Always show app frame bar so you can always drag the app
- Ensure validation text is red
- Remove onboarding background and just use black, so that red/gray text shows up appropriately
- Fixes padding of callout warning to copy recovery phrase
- Fixes border width of transaction logs
- Uses Page component more extensively for consistent headings
- Updates buttons to use ui-toolkit buttons
- Aligns create/import buttons to bottom
- Closes network drawer when interacting with the navbar
- Fixes broken 'view transactions' button
- Adjusts height of drawer

# Demo 📺

### CODE REVIEW

File: libs/wallet-ui/src/components/code-window/code-window.tsx [hunk 1]
```diff
@@ -8,16 +9,27 @@ export const CodeWindow = ({
...
 }) => {
+  const codeElRef = useRef<HTMLDivElement>(null)
+
+  // scroll to the dummy element when new logs arrive
+  // this fucks with the transaction page
+  // useEffect(() => {
+  //   if (codeElRef.current) {
+  //     codeElRef.current.scrollIntoView({ behavior: 'smooth' })
+  //   }
+  // }, [content])
```
Comment (dexturr) [inline libs/wallet-ui/src/components/code-window/code-window.tsx:20]: Delete and create an issue?

File: libs/wallet-ui/src/routes/settings/home/home.tsx [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
-  )
+  const items = [
+    {
+      title: 'App settings',
+      path: Paths.Settings.AppSettings,
+    },
+    // {
+    //   title: 'Networks',
+    //   path: Paths.Settings.Networks,
+    // },
```
Comment (notbot00) [inline libs/wallet-ui/src/routes/settings/home/home.tsx]: question (non-blocking): not sure what's going on with this commented out path, probably can be removed? 

```suggestion
```

File: libs/wallet-ui/src/components/code-window/code-window.tsx [hunk 3]
```diff
@@ -8,16 +9,27 @@ export const CodeWindow = ({
   content: ReactNode
   text: string
 }) => {
+  const codeElRef = useRef<HTMLDivElement>(null)
+
+  // scroll to the dummy element when new logs arrive
```
Comment (notbot00) [inline libs/wallet-ui/src/components/code-window/code-window.tsx]: created an issue for this, so we have some context for this: https://github.com/vegaprotocol/vegawallet-ui/issues/179

**Review by notbot00 — APPROVED** [review]

File: libs/wallet-ui/src/components/code-window/code-window.tsx (hunk 3 above)
Comment (mattrussell36) [reply libs/wallet-ui/src/components/code-window/code-window.tsx]: Good idea, I'll clear out the code too..

**Pull request merged**